### PR TITLE
Added additional intermediate learning resources

### PR DESCRIPTION
**Pull request by intellectape**
This PR contains resources related to RxJS and Protractor which are widely used in Angular Application Development and testing

### CODE REVIEW

File: Angular.md [hunk 1]
```diff
@@ -3,8 +3,19 @@
 ## For Beginners
 
 1.  [Angular 2 Book](https://goalkicker.com/Angular2Book/)
-1.  [Angular 6 Complete Playlist (45 videos)](https://www.youtube.com/playlist?list=PLC3y8-rFHvwhBRAgFinJR8KHIrCdTkZcZ)
+2.  [Angular 6 Complete Playlist (45 videos)](https://www.youtube.com/playlist?list=PLC3y8-rFHvwhBRAgFinJR8KHIrCdTkZcZ)
 
 ## For People with Intermediate Level Expertise
 
+### RXJS - Reactive Programming
```
Comment (harshitjuneja) [inline Angular.md]: Reactive programming is a general topic. Please make a separate file dedicated to Reactive programming and add there.

**Review by harshitjuneja — CHANGES_REQUESTED** [review]
Thanks for the PR. Please move reactive programming to a separate file.

**Comment by harshitjuneja** [comment]
@intellectape do you plan to pick them up?

**Comment by intellectape** [comment]
@harshitjuneja I have removed those statements from this file. I will be making a separate PR for reactive programming as this one is specific to Angular.

**Pull request merged**